### PR TITLE
Fix for Documentations being created incompletely

### PR DESCRIPTION
**Pull request by RobinTheSongbird**
.readthedocs.yaml: Requierd for Readthedocs to create documentation

conf.py: fix for issue preventing Autosummary from applying recursively, as well as changes to the html theme to avoid verion conflicts of dependencies

literature.rst: Literature.bib pointed back at ..\morty\external\literature.bib, so this was changed to directly point there

literature.bib: no longer needed due to above change

### CODE REVIEW

**Comment by carstenblt** [comment]
Awesome to see some love for this old project. I don't have it set up right now so I will need some time to check it - but it will be merged.

**Comment by RobinTheSongbird** [comment]
Made additional changes to Comments in analytical package. Readthedocs for the fork can be viewed at https://morty-test.readthedocs.io/en/latest/index.html

File: .readthedocs.yaml
Comment (RobinTheSongbird) [inline .readthedocs.yaml:1]: This file is nessecary for Readthedocs to work now

File: docs/conf.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def setup(app):
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
```
Comment (RobinTheSongbird) [inline docs/conf.py:103]: this raised a warning when generating the docs and defaulted to en, setting it to en prevents the warning.

File: docs/conf.py [hunk 2]
```diff
@@ -116,20 +116,13 @@ def setup(app):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'alabaster'
```
Comment (RobinTheSongbird) [inline docs/conf.py:120]: sphinx_rtd_theme has dependencies that conflict with the current version of bibtex, so one of sphinx's default themes was used.

File: docs/conf.py [hunk 3]
```diff
@@ -187,6 +180,7 @@ def setup(app):
 
 
 # -- Extension configuration -------------------------------------------------
+bibtex_bibfiles = ['..\morty\external\literature.bib']
```
Comment (RobinTheSongbird) [inline docs/conf.py:184]: the previous literature.bib file pointed back at this location, this seems simpler

File: docs/literature.bib
Comment (RobinTheSongbird) [inline docs/literature.bib:1]: obsolete due to changes in conf.py

File: docs/literature.rst [hunk 4]
```diff
@@ -4,6 +4,6 @@ Literature
 Here you can find all the literature cited in the doctrings of the *morty*
 package. It should ideally cover most of the theory and conventions we use.
 
-.. bibliography:: literature.bib
+.. bibliography:: ..\morty\external\literature.bib
```
Comment (RobinTheSongbird) [inline docs/literature.rst:7]: see conf.py

File: morty/analytical/_csa.py [hunk 5]
```diff
@@ -8,7 +8,7 @@
 import numpy as np
 from numpy.polynomial import legendre
 import scipy.linalg
-from scipy.ndimage.filters import gaussian_filter1d
+from scipy.ndimage import gaussian_filter1d
```
Comment (RobinTheSongbird) [inline morty/analytical/_csa.py:11]: changed to conform to current scipy version

File: morty/analytical/_csa.py [hunk 6]
```diff
@@ -32,10 +32,28 @@ def zcw(num_angles, spherical=True):
...
+
+        array([[ 0.314159, 0.      , 0.309017],
+               [ 0.942478, 2.51327 , 0.809017],
+               [ 1.5708  , 5.02655 , 1.      ],
+               [ 2.19911 , 1.25664 , 0.809017],
+               [ 2.82743 , 3.76991 , 0.309017]
+    
+    Note that the array has 5 rows, as the next higher Fibonacci number to 4 is 5.
+                       
+
```
Comment (RobinTheSongbird) [inline morty/analytical/_csa.py:56]: changed to accurately reflect the returned array and added an example

File: morty/analytical/ddevolution.py [hunk 7]
```diff
@@ -79,14 +79,19 @@ def dist_from_dd_coupling_constant(ddcc, gamma1, gamma2):
 
     Parameters
     ----------
-    r : float
-        Distance between the nuclei in m.
+    ddcc : float
+        Coupling constant in rad/s.
```
Comment (RobinTheSongbird) [inline morty/analytical/ddevolution.py:83]: this seems to have been copy and pasted from the above function. Changed params and returns to accurately describe the function

File: morty/analytical/lineshapes.py [hunk 8]
```diff
@@ -42,6 +42,9 @@ def pseudovoigt(x_axis, iso, sigma, gamma, intensity, eta):
         Pseudo Voigt Profile for given params of same length as s_axis.
 
     """
```
Comment (RobinTheSongbird) [inline morty/analytical/lineshapes.py:44]: documentation for parameter gamma was wrong, the function raised an error if gamma = None was passed. If gamma is None it is now set to the value of sigma to reflect the description given.

File: morty/analytical/spectra.py [hunk 9]
```diff
@@ -845,8 +863,8 @@ def baseline(self, f2range=((0, None),), f1range=None, deg=2):
         Baseline correction for the spectrum.
 
         Performs a polynomial fit to the spectrum and saves a corrected
-        spectrum to `Spectrum1D.spc_c` and the polynom function used as a
-        basis in `Spectrum1D.base`.
```
Comment (RobinTheSongbird) [inline morty/analytical/spectra.py:849]: this seems to have been copied and pasted from Spectrum1D.

File: morty/analytical/spectra.py [hunk 10]
```diff
@@ -1114,7 +1136,7 @@ def integrate_deconvoluted(self, signals, spc_slice=None, start_spc=0,
 
         Parameters
         ----------
-        signals : list of list of tuples
```
Comment (RobinTheSongbird) [inline morty/analytical/spectra.py:1117]: changed to have consistent wording with Spectrum1D.integrate_deconvoluted()

File: docs/conf.py (hunk 1 above)
Comment (Mnhrt) [reply docs/conf.py:103]: GJ

File: morty/analytical/_csa.py (hunk 6 above)
Comment (Mnhrt) [reply morty/analytical/_csa.py:56]: Good job.

**Comment by Mnhrt** [comment]
Necessary changes in comment lines :)

**Pull request merged**